slicer/SemiologyVisualization.py

```diff
@@ -131,7 +131,7 @@ def getInclusionsWidget(self):
       'Invasive EEG recording and/or electrical stimulation, mapping seizure semiology'
     )
     self.concordanceCheckBox.setToolTip(
-      'Multimodal concordance between brain imaging and non-invasive neurophysiology'
+      'Multimodal concordance between brain imaging and neurophysiological'
```

Comment:
Why not just "neurophysiology"?

Reply:
because it had findings. edited as to your preference!